elixir_daisy/settings_local.template.py

```diff
@@ -6,9 +6,24 @@
     'guardian.backends.ObjectPermissionBackend',
 ]
 
+# SECURITY WARNING: change the secret key used in production and keep it secret !
+SECRET_KEY = 'qe1lmt43v1n%66vibs0&0s9qw7i!xjs^!i#f#$t_7-r8n&=+sp'
```

Comment:
I would be slightly more in favour of:

```
L09:  # SECURITY WARNING: change the secret key used in production and keep it secret !
L10:  SECRET_KEY = None  # Generate a secret key by e.g. django.core.management.utils.get_random_secret_key()
L11:  if SECRET_KEY is None: rase NotImplementedError('You must specify SECRET_KEY in settings_local.py')
```

Reply:
This way:
a) if the user downloads the code, tries to run it and does nothing - he will see an error with a direct message
b) if the user changes the value, the exception will not be raised

Reply:
that looks good! :) I have added the change